### PR TITLE
Add cn.vu

### PR DESCRIPTION
**Pull request by neupeer**
Add new private domain cn.vu

* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: http://www.com.vc
www.com.vc provides web hosting and email services. I am the founder and director.

Reason for PSL Inclusion
====

As we have no control over what the user serves on these subdomains, we're looking to get them added to the PSL, improving the cookie security and browser's behavior.

DNS Verification via dig
=======

```
dig +short TXT _psl.cn.vu
"https://github.com/publicsuffix/list/pull/987"
```

make test
=========

Test run and passed.

### CODE REVIEW

**Comment by dnsguru** [comment]
@neupeer Li can you submit this with a different email address - I have challenges with bounced messages frequently on @qq.com email addresses and being able to contact listing managers is often important.

**Comment by neupeer** [comment]
lihui@sinopub.com
Thanks for your reminder!

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12963,6 +12963,10 @@ hk.org
 ltd.hk
 inc.hk
 
+// www.com.vc : http://www.com.vc
+// Submitted by Li Hui <214274110@qq.com>
```
Comment (dnsguru) [inline public_suffix_list.dat:12967]: Please update to a non qq.com email address and re-submit

**Comment by dnsguru** [comment]
@weppos could use a second opinion could you please also review this?

**Comment by neupeer** [comment]
@weppos Okay, you can check this website, the subdomain will be allocated to companies in China.
www.zaozhuang.cn.vu

**Pull request merged**